Microsoft.Toolkit.Uwp.SampleApp/Controls/SampleAppMarkdownRenderer.cs

```diff
@@ -34,6 +36,7 @@ internal class SampleAppMarkdownRenderer : MarkdownRenderer
         public SampleAppMarkdownRenderer(MarkdownDocument document, ILinkRegister linkRegister, IImageResolver imageResolver, ICodeBlockResolver codeBlockResolver)
             : base(document, linkRegister, imageResolver, codeBlockResolver)
         {
+            LanguageRequested += SampleAppMarkdownRenderer_LanguageRequested;
```

Comment:
This needs to be unregistered somewhere too, eh?

Reply:
Is this necessary?

https://stackoverflow.com/questions/506092/is-it-necessary-to-explicitly-remove-event-handlers-in-c-sharp

I'm not sure how easy this would be, I might need some custom event handler to remove all listeners, but I thought when the MarkdownTextBlock gets destroyed, all exterior refrences are removed so it can delete the object?

There isn't any current way to do this yet.

Reply:
Ah, should be fine then if you haven't observed a leak.  I've been dealing with a lot of registrations on other objects for Monaco, so I've been cleaning mind up. :)